### DISCUSSION TITLE
spellcheckjs written in python

### DISCUSSION THREAD

**Discussion by imdevarsh** [SpellCheckJS/SpellCheckJS#12] [General]
According to github, spellcheckjs is in python due to the CI.

**Comment by hello-smile6** [comment]
> According to github, spellcheckjs is in python due to the CI.

Kinda ironic. Can I fix that?

**Reply by imdevarsh** [reply]
Put the CI in a separate repo.

**Reply by hello-smile6** [reply]
> Put the CI in a separate repo.

Okay. Could you do it for me, if I gave you access?

**Reply by imdevarsh** [reply]
Just make a new repo with an action, and use the action. Like how @scratch-advancedtopics does it.

**Reply by hello-smile6** [reply]
> Just make a new repo with an action, and use the action. Like how @scratch-advancedtopics does it.

Okay.

**Reply by hello-smile6** [reply]
@webdev03 If I do that, how do I run the script?

**Reply by imdevarsh** [reply]
I'll do it.

**Reply by hello-smile6** [reply]
> I'll do it.

Okay, I gave you access to my CI repo.
